.circleci/config.yml

```diff
@@ -330,18 +368,54 @@ jobs:
             done
 
   # defines the qa deployment
-  # Will fetch the `github/ConsenSys/quorum-key-manager-kubernetes` chart using
+  # Will update the existing TLS secret of the QKM with fresh PKI data
+  # Needs AWS authentication + kubectl tool
+  # Persists CA + certs + keys in the workspace for e2e job tu use
+  # Will fetch the `github/ConsenSys/quorum-key-manager-kubernetes` chart using the following
   # QKM_TAG=${CIRCLE_SHA1:0:7}
   # QKM_REPOSITORY=${CLOUDSMITH_DEV_REPO}/quorum-key-manager
   # REGISTRY_USERNAME, docker registry user
   # REGISTRY_PASSWORD, docker registry pw
   # see scripts/deploy-remote-env.sh for details
   deploy-qa:
-    docker:
-      - image: cimg/base:2021.12
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: medium
+    parameters:
+      cluster-name:
+        type: string
+        default: "orchestrate"
+      namespace:
+        type: string
+        default: "qa-qkm"
     steps:
+      - run:
+          name: Uninstall Go
+          command: sudo rm -rvf /usr/local/go/
+      - go/install:
+          version: 1.16.8
       - checkout
       - set_remote_environment_variables
+      - prepare_golang
+      - kubernetes/install
+      - aws-eks/update-kubeconfig-with-authenticator:
+          cluster-name: << parameters.cluster-name >>
+      - run:
+          name: Renew Qkm TLS Secret
```

Comment:
IMO, and align to what @darioAnongba raise the other day, we should not generate or depend on pipeline values to deploy our QA/Prod env, they already have a pre define setup and we only trigger the upgrade using the latest image from here, otherwise we are making our CI depending on Helms constrains which might change over time

Reply:
I am not sure I get what you expect here, these are dependencies needed to access the cluster, that specific part is here to align the qkm existing secret (TLS stuff) with the deployed values. There is no other way I can do that.

Reply:
isn't it possible to mainta same TLS configuration over different deployment?